src/material/form-field/form-field.scss

```diff
@@ -63,10 +63,14 @@
   width: 100%;
 }
 
-// The MDC text-field should stretch to the width of the host `<mat-form-field>` element.
-// This allows developers to control the width without needing custom CSS overrides.
 .mat-mdc-text-field-wrapper {
+  // The MDC text-field should stretch to the width of the host `<mat-form-field>` element.
+  // This allows developers to control the width without needing custom CSS overrides.
   width: 100%;
+  // MDC applies the `.mdc-text-field--disabled` class to this element when the input is disabled.
```

Comment:
Nit question: We don't use foundations anymore, so it's ourselves adding the class right? Maybe the comment should say that MDC's CSS sets `pointer-events: none` for the whole wrapper and the wrapper includes prefixes/suffixes in our form-field implementation?

Reply:
We're adding the class, but the `pointer-events: none` is coming from MDC's CSS. I don't think just leaving it and documenting is the correct thing to do. If the user really does want the prefix/suffix disabled when the control is disabled, then they should tie together the disabled states in their code. If we just slap a `pointer-events: none` on it it becomes unclickable, but people can still navigate to and activate it via keyboard

Reply:
Sorry, I was not saying you should revert this PR 😅 More I felt like the current comment made it sound like MDC applies the `.mdc-text-field--disabled` class and we don't have control over it. In reality _we_ apply the class, but MDC's CSS sets `pointer-events:none`.

Reply:
Ah gotcha, yeah I'll reword that